tests/validation/tests/v3_api/test_bkp_restore_local.py

```diff
@@ -102,10 +98,7 @@ def test_rbac_bkp_restore_restore(role):
     user_token2 = rbac_get_user_token_by_role(role)
     user_client2 = get_client_for_token(user_token2)
     user_cluster2 = user_client2.list_cluster(name=CLUSTER_NAME).data[0]
-    with pytest.raises(ApiError) as e:
-        user_cluster2.restoreFromEtcdBackup(etcdBackupId=backup_id)
-    assert e.value.error.status == 403
-    assert e.value.error.code == 'PermissionDenied'
+    assert hasattr(user_cluster2, 'restoreFromEtcdBackup') == False
```

Comment:
This is a good call, im just not sure if this would work on 2.5.x

Reply:
I just updated the PR, this should take care of the difference in rancher v2.5.x and v2.6.x